tests/data/test_preprocess_cord.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for making datasets for contradictory-claims."""
+
+# -*- coding: utf-8 -*-
+
+import os
+import unittest
+from datetime import datetime
+
+# from contradictory_claims.data.preprocess_cord import filter_metadata_for_covid19
+
+from .constants import metadata_path, pub_date_cutoff, virus_lex_path
+
+
+class TestPreprocessCord(unittest.TestCase):
+    """Tests for preprocessing CORD-19."""
+
+    def test_find_files(self):
+        """Test that input files are found properly."""
+        self.assertTrue(os.path.isfile(metadata_path),
+                        "Metadata.csv not found at {}".format(metadata_path))
+
+        self.assertTrue(os.path.isfile(virus_lex_path),
```

Comment:
Already being tested in another test function right?

Reply:
Yeah, I was somehow uncertain about whether to keep the test case for virus words in both places and decided to keep it for the time being...But I see it's redundant and not very necessary so I've removed it now. Test for metadata.csv remains.

Reply:
Yeah, I wasn't sure whether to keep the virus words test or not since it was already tested and decided to keep it..But I see it's redundant and can potentially be confusing, so I've removed it now.